code/modules/food_and_drinks/recipes/tablecraft/recipes_lizard.dm

```diff
@@ -22,11 +22,16 @@
 /datum/crafting_recipe/food/shredded_lungs
 	name = "Crispy shredded lung stirfry"
 	reqs = list(
+		/obj/item/reagent_containers/cup/bowl = 1,
 		/obj/item/organ/internal/lungs = 1,
 		/obj/item/food/grown/onion = 1,
 		/obj/item/food/grown/carrot = 1,
 		/obj/item/food/grown/chili = 1
```

Comment:
is this something that can be alphabetized?

Reply:
I was hoping that the order would be the one that was displayed in the recipe manager, because having bowl first makes logical sense, but I don't think it passes through, so I will alphabetize it.